controllers/imageclusterinstall_controller.go

```diff
@@ -190,47 +190,14 @@ func (r *ImageClusterInstallReconciler) Reconcile(ctx context.Context, req ctrl.
 			log.WithError(err).Error("failed to set ClusterDeployment metadata")
 			return ctrl.Result{}, err
 		}
-	}
-
-	return ctrl.Result{}, nil
-}
-
-func (r *ImageClusterInstallReconciler) setImageReadyCondition(ctx context.Context, ici *v1alpha1.ImageClusterInstall, err error) error {
-	cond := metav1.Condition{
-		Type:    v1alpha1.ImageReadyCondition,
-		Status:  metav1.ConditionTrue,
-		Reason:  v1alpha1.ImageReadyReason,
-		Message: v1alpha1.ImageReadyMessage,
-	}
-
-	if err != nil {
-		cond.Status = metav1.ConditionFalse
-		cond.Reason = v1alpha1.ImageNotReadyReason
-		cond.Message = err.Error()
-	}
-
-	patch := client.MergeFrom(ici.DeepCopy())
-	meta.SetStatusCondition(&ici.Status.ConfigConditions, cond)
-	return r.Status().Patch(ctx, ici, patch)
-}
-
-func (r *ImageClusterInstallReconciler) setHostConfiguredCondition(ctx context.Context, ici *v1alpha1.ImageClusterInstall, err error) error {
-	cond := metav1.Condition{
-		Type:    v1alpha1.HostConfiguredCondition,
-		Status:  metav1.ConditionTrue,
-		Reason:  v1alpha1.HostConfiguraionSucceededReason,
-		Message: v1alpha1.HostConfigurationSucceededMessage,
-	}
 
-	if err != nil {
-		cond.Status = metav1.ConditionFalse
-		cond.Reason = v1alpha1.HostConfiguraionFailedReason
-		cond.Message = err.Error()
+		if err := r.setClusterInstalledConditions(ctx, ici); err != nil {
```

Comment:
Seems premature but I guess for the time being we can't really tell so why not

Reply:
I'm actually also already setting the CD to installed as well so I figure we can change both later if it becomes a problem and we want to implement some kind of monitoring

ref: https://github.com/openshift/image-based-install-operator/blob/6f767d683a7091a4831fe27b8b034a79d4f54e8e/controllers/imageclusterinstall_controller.go#L784